wpeverest/user-registration
# UR-596 Tweak - Value Based Review Notice.

### All Submissions:






### Changes proposed in this Pull Request:

In this PR if user completes there first 20 users then Review notice will appears.

Closes https://themegrill.atlassian.net/browse/UR-596

### How to test the changes in this Pull Request:

1. To get review notice user have to spend atleast 7 days activation of plugin.
2. Once User Dismissed Notice Permanently then review notice not appears again.
3. After 7 days if user completes 20 users from user registration plugin then user will get review notice with different content.

### Types of changes:




### Other information:




### Changelog entry

> Tweak - Value Based Review Notice.
